aggregate_spatial_binary.json

```diff
@@ -65,15 +59,15 @@
         }
     ],
     "returns": {
-        "description": "A vector data cube with the computed results. The vector data cube implicitly gets restricted to the bounds of the polygons as if ``filter_polygon()`` would have been used with the same values for the corresponding parameters immediately before this process.\n\nThe computed value is stored in dimension with the name that was specified in the parameter `name`.\n\nThe computation also stores information about the total count of pixels (valid + invalid pixels) and the number of valid pixels (see ``is_valid()``) in each geometry. These values are stored as attributes of the result value with the attribute names `total_count` and `valid_count`.",
+        "description": "A vector data cube with the computed results. The vector data cube implicitly gets restricted to the bounds of the geometries as if ``filter_spatial()`` would have been used with the same values for the corresponding parameters immediately before this process.\n\nThe computed value is stored in dimension with the name that was specified in the parameter `target_dimension`.\n\nThe computation also stores information about the total count of pixels (valid + invalid pixels) and the number of valid pixels (see ``is_valid()``) for each geometry. These values are stored as new dimension with a dimension name derived from `target_dimension` by adding the suffix `_meta`. The new dimension has the dimension labels `total_count` and `valid_count`.",
         "schema": {
             "type": "object",
             "subtype": "vector-cube"
         }
     },
     "exceptions": {
         "TooManyDimensions": {
-            "message": "The number of dimensions must be reduced to three for 'aggregate_polygon'."
+            "message": "The number of dimensions must be reduced to three for 'aggregate_spatial'."
```

Comment:
This should be `aggregate_spatial_binary`?

Reply:
Indeed, fixed.